websites/S/Spinsha.re/dist/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.0",
+    "author": {
+      "name": "ayanamy.jy126",
+      "id": "243279723487035393"
+    },
+    "service": "SpinShare",
+    "description": {
+      "en": "SpinShare is a chart sharing service for Spin Rhythm XD."
+    },
+    "url": "spinsha.re",
+    "regExp": "(spinsha[.]re)[/]",
```

Comment:
Is there a good reason for this? Otherwise you could just get away with just the URL.

Reply:
o okay thanks